### ISSUE TITLE
bun:jsc being flagged as required node:protocol

### ISSUE THREAD

**Issue by noahehall** [sindresorhus/eslint-plugin-unicorn#2226]
~same as this: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1931~
```ts

jsc = await import('bun:jsc')
// is converted to this
jsc = await import('node:bun:jsc')
```

```js
const code = 'that should be ok';
```

**Comment by fisker** [comment]
~same as this: #1931~

Apparently, it's not.

**Comment by mangs** [comment]
@fisker I get this same error with `bun:test` too (Bun's test runner); it expects `node:bun:test` which doesn't make any sense.

I'm not really sure `bun:` is a protocol, rather it's kind of a namespace, kind of not. For example, here are many of Bun's native APIs: https://bun.sh/docs/runtime/bun-apis

Let's use the `Glob` API as an example. As you can see, it's written as `Bun.Glob` implying that it only exists as a member of the `Bun` global variable. However, if you click into the details of the Glob API (https://bun.sh/docs/api/glob), you can see it can also be used as an import: `import { Glob } from "bun";`. In other words, accessing APIs via imports or via the global `Bun` variable are interchangeable. So this ESLint plugin should not be treating `bun`, `bun:test`, `bun:jsc`, or any other Bun imports as being Node.js imports.
